common/djangoapps/track/shim.py

```diff
@@ -83,6 +83,13 @@ def remove_shim_context(event):
     'edx.video.seeked': 'seek_video',
     'edx.video.transcript.shown': 'show_transcript',
     'edx.video.transcript.hidden': 'hide_transcript',
+
+    # Needs to add the following, but 'event' is seen as a str with them
+    # Mobile team?
+    # 'edx.video.closed_captions.shown': 'edx.video.closed_captions.shown',
+    # 'edx.video.closed_captions.hidden': 'edx.video.closed_captions.hidden',
+    # 'edx.video.language_menu.shown': 'video_show_cc_menu',
+    # 'edx.video.language_menu.hidden': 'video_hide_cc_menu',
```

Comment:
@mulby I'd like to have the correct `name` event for mobile even if we can't update the JS at this time. When I add these in, the passed `event` is seen as a string and things break, despite following the examples with dot notation above. What I'm going for here is:

* we're keeping the `video_show/hide_cc_menu` `event_type`'s, but adding a `name` that uses the updated namespaced varieties.
* adding new `name` and `event_type` for closed-captions.

Do you think you could help with this, or is there someone you suggest who might be more suitable?